cloud-regionsrv-client.spec

```diff
@@ -29,6 +35,8 @@ Source0:        %{name}-%{version}.tar.bz2
 Patch0:         fix-for-sles12-disable-ipv6.patch
 # PATCH-FIX-SLES12 fix-for-sles12-disable-registry.patch
 Patch1:         fix-for-sles12-disable-registry.patch
+# PATCH-FIX-SLES12 fix-for-sles12-no-trans_update.patch
+Patch2:         fix-for-sles12-no-trans_update.patch
```

Comment:
The patch file `fix-for-sles12-no-trans_update.patch` is not part of the git repo. Can you please add it like the other patch files. Thanks

Reply:
I tested the package build for all maintained targets. One thing I noted is that for SLE15 the package builds against python 3.6, I guess this is intentional ?